### DISCUSSION TITLE
Why does the IG Downloader section occasionally stop working and then resume working?

### DISCUSSION THREAD

**Discussion by NomanBaigA** [ashutoshkrris/EazyLoader#72] [General]
Is it because of instaloader module? or something else  ||  I reported it before as a bug "/EazyLoader/issues/70"

**Comment by ashutoshkrris** [comment]
Hi @NomanBaigA 
Yes, it's because of the Instaloader module.
Instaloader requires an Instagram account for most of the functionalities. The account being used by EazyLoader gets blocked by Instagram team, and hence Instagram downloader stops working.

**Reply by NomanBaigA** [reply]
Ok, Thanks 👍🏻
